Examples/Nodify.Playground/ISettingViewModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Nodify.Playground
+{
+    public enum SettingsType
+    {
+        Boolean,
+        Number,
+        Option, //For ComboBox
+        Point
+    }
+
+    public interface ISettingViewModel
+    {
+        string Name { get; }
+        string? Description { get; } // This is the tooltip
```

Comment:
Please write comments in the correct format if you think they are necessary. (see https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments)

Reply:
The comment can be written like this:

```xml
/// <summary>
/// Represents the content within the tooltip.
/// </summary>
```